transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/Hl7Page.java

```diff
@@ -199,11 +199,11 @@ private void updateSettingsBasedOnFilePath(String path) throws Exception {
     @Override
     public void createControl(final Composite parent) {
         if (this.isSource) {
-            setTitle("Source Type (JSON)");
-            setDescription("Specify details for the source JSON for this transformation.");
+            setTitle("Source Type (JSON)"); //$NON-NLS-1$
+            setDescription("Specify details for the source JSON for this transformation."); //$NON-NLS-1$
```

Comment:
Please ignore all of these in the HL7Page.  This is just a placeholder right now and is not used by the code.  It was created a while ago in preparation for when we have first-class HL7 support.  The code will change significantly then, so it’s not worth addressing now.

> On Aug 18, 2016, at 10:45 AM, Aurélien Pupier <notifications@github.com> wrote:
> 
> In transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/Hl7Page.java <https://github.com/fusesource/fuseide/pull/635#discussion_r75333830>:
> 
> > @@ -199,11 +199,11 @@ private void updateSettingsBasedOnFilePath(String path) throws Exception {
> >      @Override
> >      public void createControl(final Composite parent) {
> >          if (this.isSource) {
> > -            setTitle("Source Type (JSON)");
> > -            setDescription("Specify details for the source JSON for this transformation.");
> > +            setTitle("Source Type (JSON)"); //$NON-NLS-1$
> > +            setDescription("Specify details for the source JSON for this transformation."); //$NON-NLS-1$
> better to externalize the description
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/fusesource/fuseide/pull/635/files/9a79f9f230b22742c5957821eae8ca4e69997505#r75333830>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAarXU5K_9wXlsMcdhmdbTEPQInzOukeks5qhH4WgaJpZM4JmHoy>.
>